src/puppetlabs/dujour/version_check.clj

```diff
@@ -163,7 +164,8 @@
                                 (log/warn e "Error occurred while checking for updates")
                                 (throw e)))]
         (if-not (nil? callback-fn)
-          (callback-fn server-response))))))
+          (callback-fn server-response)
+          server-response)))))
```

Comment:
Was this change intentional? I guess if no callback function is defined it just tries to run version-check?